CTOR - 815 : add(network::cisco::standard::snmp): add option --add-fc-fe-errors (#3620)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md

</TabItem>
<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">

| Alias         | Modèle de service                            | Description                                                   | Découverte |
|:--------------|:---------------------------------------------|:--------------------------------------------------------------|:----------:|
| Aaa-Servers   | Net-Cisco-Standard-Aaa-Servers-SNMP-custom   | Contrôle les serveurs AAA                                     | X          |
| Anycast       | Net-Cisco-Standard-Anycast-SNMP-custom       | Contrôle le type de trafic                                    |            |

> | Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Contrôle la table arp                                         |            |

Reviewer: ```suggestion
| Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Contrôle la table ARP                                         |            |
```

|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Cisco-Standard-SNMP-custom** |

Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.

#### Découverte de services

| Nom de la règle                         | Description                                                             |
|:----------------------------------------|:------------------------------------------------------------------------|
| Net-Cisco-Standard-SNMP-Aaa-Server-Name | Découvre les serveurs AAA et supervise l'utilisation                                                                        |

> | Net-Cisco-Standard-SNMP-Interface-Name  | Découvre les interfaces réseaux et supervise le statut et l'utilisation |

Reviewer: ```suggestion
| Net-Cisco-Standard-SNMP-Interface-Name  | Découvre les interfaces réseau et supervise leur statut et leur utilisation |
```

<Tabs groupId="sync">
<TabItem value="Aaa-Servers" label="Aaa-Servers">

| Macro                            | Description                                                                                                                                      | Valeur par défaut    | Obligatoire |
|:---------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|

> | FILTERNAME                       | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                         |                      |             |

Reviewer: ```suggestion
| FILTERNAME                       | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                         |                      |             |
```

| CRITICALCOUNTVOLTAGE     | Thresholds                                                                                                                                        |                                                                      |             |
| EXTRAOPTIONS             | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).  | --verbose  --filter-perfdata='^(sensor\.(celsius\_\|rpm\_)\|temp\_)' |             |

</TabItem>
<TabItem value="Hsrp" label="Hsrp">

| Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |

Reviewer: ```suggestion
| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |
```

Reviewer: the sentences make no sense

> | ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |

Author: ```suggestion
| ROLE         | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states.                      | primary           | X           |
```

> | ROLE         | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states.                      | primary           | X           |

Reviewer: ```suggestion
| ROLE         | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state.                      | primary           | X           |
```

> | ROLE         | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states.                      | primary           | X           |

Author: ```suggestion
| ROLE         | If role is 'primary', an error is returned when HSRPs are in a 'standby' states. If role is 'secondary', an error is returned when HSRPs are in a 'active' states.                      | primary           | X           |
```

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose         |             |

</TabItem>
<TabItem value="Interfaces" label="Interfaces">

| Macro                   | Description                                                                                                                                                              | Valeur par défaut                                     | Obligatoire |
|:------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | UNITSTRAFFIC            | Units of thresholds for the traffic ('percent\_delta', 'bps', 'counter')                                                                                                 | percent\_delta                                        |             |

Reviewer: ```suggestion
| UNITSTRAFFIC            | Units in which the thresholds are given for the traffic ('percent\_delta', 'bps', 'counter')                                                                                                 | percent\_delta                                        |             |
```

| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
| --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='auth'                                                                                                                                                            |

> | --filter-name            | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |

Reviewer: ```suggestion
| --filter-name            | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
```

| --show-cache                                    | Display cache interface data.                                                                                                                                                                                                                                                                                                                                               |

</TabItem>
<TabItem value="Arp" label="Arp">

| Option                   | Description                                                                      |
|:-------------------------|:---------------------------------------------------------------------------------|
| --filter-macaddr         | Filter mac addresses (can be a regexp).                                          |

> | --filter-ipaddr          | Filter ip addresses (can be a regexp).                                           |

Reviewer: ```suggestion
| --filter-ipaddr          | Filter IP addresses (can be a regexp).                                           |
```

| Option        | Description                                                                                                                                         |
|:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
| --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |

> | --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |

Reviewer: ```suggestion
| --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```

Reviewer: same here, doesn't make sense

> | --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |

Author: ```suggestion
| --role        | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states. (default: 'primary')    |
```

> | --role        | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states. Default: 'primary'.    |

Reviewer: ```suggestion
| --role        | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state. Default: 'primary'.    |
```

| --warning-usage  | Warning threshold in percent.                                                                                                                                                              |
| --critical-usage | Critical threshold in percent.                                                                                                                                                             |
| --filter-pool    | Filter pool to check (can use regexp).                                                                                                                                                     |

> | --check-order    | Check memory in standard cisco mib. If you have some issue (wrong memory information in a specific mib), you can change the order (default: 'enhanced\_pool,pool,process,system\_ext').    |

Reviewer: ```suggestion
| --check-order    | Check memory in standard cisco MIB. If you have some issue (wrong memory information in a specific MIB), you can change the order (default: 'enhanced\_pool,pool,process,system\_ext').    |
```

| --critical-status | Define the conditions to match for the status to be CRITICAL (default: '%{op\_status} =~ /up/ && %{state} =~ /blocking\|broken/'). You can use the following variables: %{state}, %{op\_status}, %{admin\_status}, %{port}, %{index}.    |

</TabItem>
<TabItem value="Stack" label="Stack">

| Option                   | Description                                                                                                                                                                                                                                                                                                                                                                                            |
|:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --warning-* --critical-* | Set thresholds for members count for each states.can be: 'waiting', 'progressing', 'added', 'ready', 'sdm-mismatch', 'version-mismatch', 'feature-mismatch', 'new-master-init', 'provisioned', 'invalid', 'removed')                                                                                                                                                                                 |

Reviewer: ```suggestion
| --warning-* --critical-* | Set thresholds on members count for each states. (Can be: 'waiting', 'progressing', 'added', 'ready', 'sdm-mismatch', 'version-mismatch', 'feature-mismatch', 'new-master-init', 'provisioned', 'invalid', 'removed')                                                                                                                                                                                 |
```
